Makefile

```diff
@@ -13,6 +6,16 @@ pip := $(virtualenv_bin)/pip
 run_dir := $(realpath $(makefile_dir)/..)/.run
 conf_dir := $(realpath $(makefile_dir)/..)/conf
 
+export PATH := $(wrapper_bin):$(virtualenv_bin):$(PATH)
+
+# Keep the user's current shell (say bash or zsh) instead of using the system's
+# default /bin/sh shell which can differ (e.g. dash by default on Ubuntu).
+OUT_SHELL := $(shell env | grep '^SHELL=' | cut -d '=' -f '2' )
+SHELL := env PATH=$(PATH) $(OUT_SHELL)
+
+python_version_full := $(wordlist 2,4,$(subst ., ,$(shell python3 --version 2>&1)))
```

Comment:
What about keeping thoses line at their initial place and avoid thoses PR diff?

Reply:
set that way it makes other bellow lines use the `shell` defined and the `virtualenv` as well